.travis.yml

```diff
@@ -0,0 +1,12 @@
+before_script: 
+- cd server
+- npm install codeclimate-test-reporter istanbul -g
+language: node_js
+node_js:
+   - "stable"
+addons:
+   code_climate:
+     repo_token: a315018021e648cae168bcf040a0809f041d9806
```

Comment:
This should not be here. Please look through how to put this in an environment variable on Travis. Please never push sensitive data to github.

Reply:
@OsayamenOsaretin done